windows/installer_jack.nsi

```diff
@@ -0,0 +1,444 @@
+; Jamulus NSIS Installer with Modern User Interface
+; Installer for Jamulus with support for JACK on Windows
```

Comment:
How much of this is identical to the standard installer?  Can those parts be factored out and !included here?

Reply:
Main difference are the custom ASIO installed check dialogs, which have been replaced by custom JACK installed check dialogs. I believe we should be able to pass on the buildoption into this script as well and use conditions in the script to enabled/disable some parts. The reason why a separate file was used was mainly to keep it simple/easy and clear.

Reply:
I'm worried where things that _should_ be common are separated and that commonality is no longer apparent.

I don't mind having things separate if it makes the behaviour easier to achive but I'd like the commonality clear - preferably by having it stored in a common file.

Reply:
Yes, I also [wondered why we need two files](https://github.com/jamulussoftware/jamulus/pull/1829#discussion_r645861615). If possible, we should unify them.

Reply:
I have checked now and it is actually not that complex to use 1 nsi installer file. So will update to use just 1 installer.nsi.
The installer.nsi will include a buildoption value, so based on that different actions can be performed in the installer.